cleanlab/filter.py

```diff
@@ -398,6 +403,194 @@ class 0, 1, ..., K-1.
     return label_issues_mask
 
 
+def _find_label_issues_multilabel(
+    labels: list,
+    pred_probs: np.ndarray,
+    return_indices_ranked_by: Optional[str] = None,
+    rank_by_kwargs={},
+    filter_by: str = "prune_by_noise_rate",
+    frac_noise: float = 1.0,
+    num_to_remove_per_class: int = None,
+    min_examples_per_class=1,
+    confident_joint: Optional[np.ndarray] = None,
+    n_jobs: int = None,
+    verbose: bool = False,
+) -> np.ndarray:
+    """
+    Parameters
```

Comment:
In the docstring for this helper method, for any arg where you just copied its docstring from the main `find_label_issues` and this arg is no different in multi-label vs regular classification settings, then let's just have the arg's docstring say:

Refer to documentation for this argument in `filter.find_label_issues()` for details.

Reply:
ok